src/PokemonApp.js

```diff
@@ -0,0 +1,22 @@
+import {Component} from 'react';
+import {CustomPaginationActionsTable} from './components/table/PokemonsTable.component';
+import {ErrorBoundary} from "./error/ErrorBoundary";
+
+class PokemonApp extends Component {
+    constructor() {
+        super();
+        this.state = {
+            pokemons: []
+        };
+    }
+
+    render() {
+        return (
+            <ErrorBoundary>
+                <CustomPaginationActionsTable pokemons={this.state.pokemons}/>
```

Comment:
I think you do not need pokemons state in this component because you create another one in _CustomPaginationActionsTable_.